test/server_testing_tools.h

```diff
@@ -174,16 +171,7 @@ class ServerTest : public ::testing::Test
     zfs1_.reset(new ZimFileServer(SERVER_PORT, options, ZIMFILES));
   }
 
-  ZimFileServer& taskbarlessZimFileServer()
-  {
-    if ( ! taskbarlessZfs_ ) {
-      taskbarlessZfs_.reset(new ZimFileServer(SERVER_PORT+1, ZimFileServer::NO_TASKBAR, ZIMFILES));
```

Comment:
This was the only place where we used `NO_TASKBAR`, so we should remove its definition in the enum.

Reply:
Good catch. Fixed by moving the commit that drops `taskbarlessZimFileServer()` before the commit that introduces `ZimFileServer::Options`.